docs/vizier/vizier.rst

```diff
@@ -128,15 +157,15 @@ Query an object
 
 For instance to query Sirius across all catalogs:
 
-.. code-block:: python
+.. doctest-remote-data::
 
     >>> from astroquery.vizier import Vizier
     >>> result = Vizier.query_object("sirius")
-    >>> print(result)
-    TableList with 275 tables:
-       '0:METAobj' with 5 column(s) and 5 row(s)
-       '1:ReadMeObj' with 5 column(s) and 5 row(s)
-       '2:I/34/greenw2a' with 16 column(s) and 1 row(s)
+    >>> print(result) # doctest:+ELLIPSIS
```

Comment:
ELLIPSIS should work out of the box. Did you find the contrary? (if yes, then we should propagate a bug report upstream, or at least to ourself as a reminder to check the configs)